tests/models/reading_comprehension/bidaf_test.py

```diff
@@ -46,7 +46,7 @@ def test_forward_pass_runs_correctly(self):
     # `masked_softmax`...) have made this _very_ flaky...
     @flaky(max_runs=5)
     def test_model_can_train_save_and_load(self):
-        self.ensure_model_can_train_save_and_load(self.param_file)
+        self.ensure_model_can_train_save_and_load(self.param_file, 2e-5)
```

Comment:
Can you make this `tolerance=2e-5`, so it's more obvious what this magic number you're passing around is?